extension/lib/shopgate/read_model/addProductsToCart.js

```diff
@@ -1,16 +1,3 @@
 /**
  *  @typedef {Object} AddProductsToCartResponse
- *  @property {string} cartId - bigcommerce cart ID
```

Comment:
you will need to pull the latest of the branch, I fixed this in #48

Reply:
@awesselburg somehow this file is completely removed, it should be still there, just not capitalized letter

Reply:
@awesselburg still not resolved